azure-pipelines.yml

```
@@ -259,4 +268,9 @@ extends:
           - output: pipelineArtifact
             displayName: 'Publish Artifact: drop'
             targetPath: "$(build.ArtifactStagingDirectory)/build"
-            artifactName: drop
\ No newline at end of file
+            artifactName: drop
+
+      - template: pipelines/templates/checkout-and-copy-1es.yml@ReleasePipelines
+        parameters:
+          directory: 'microsoft-graph-explorer-v4'
+          repoName: ReleasePipelines
```

If we add this dependsOn parameter, the job will wait until job Three is done and use it to create a consolidated drop

```suggestion
          repoName: ReleasePipelines
          dependsOn: ['Three']

```